### PR TITLE
Add Fluid support for AE2

### PR DESCRIPTION
**Pull request by fionera**
A quick implementation of #247

### CODE REVIEW

File: src/main/java/org/squiddev/plethora/integration/appliedenergistics/MetaAppliedEnergistics.java [hunk 1]
```diff
@@ -67,6 +69,8 @@ public IAEItemStack getExample() {
 		}
 	};
 
+	private static final MetaFluidStack META_FLUID_STACK = new MetaFluidStack();
```
Comment (SquidDev) [inline src/main/java/org/squiddev/plethora/integration/appliedenergistics/MetaAppliedEnergistics.java]: Can you refactor the `MetaFluidStack.getMeta` into a static method (which the instance method just delegates too)? Just saves us having to instantiate it here.

**Review by SquidDev — CHANGES_REQUESTED** [review]
Thank you very much for adding this! I know Plethora isn't the easiest codebase to understand.

I've just left one minor nitpick, but otherwise it looks good. Thank you!

**Review by SquidDev — APPROVED** [review]

**Comment by SquidDev** [comment]
Merged. Thank you very much! ❤️

**Comment by pipe0482** [comment]
I was playing with your mod and I love it, however I needed to query my system for fluids and found that it's not supported. I found this PR but noticed it was merged shortly after the last release, so are there any plans of releasing a new version? Thanks!

EDIT: I got a build working, so I don't need it as much now 😄

**Pull request merged**